app/models/manageiq/providers/vmware/cloud_manager.rb

```diff
@@ -31,6 +31,90 @@ def self.description
     @description ||= "VMware vCloud".freeze
   end
 
+  def self.params_for_create
+    @params_for_create ||= {
+      :title  => "Configure #{description}",
+      :fields => [
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.server",
+          :label      => "Server Hostname/IP Address",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
+        },
+        {
+          :component => "text-field",
+          :name      => "endpoints.default.port",
+          :label     => "Port",
+          :type      => "number",
+        },
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.username",
+          :label      => "Username",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
+        },
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.password",
+          :label      => "Password",
+          :type       => "password",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
+        },
+        {
+          :component => "text-field",
+          :name      => "endpoints.events.server",
+          :label     => "AMQP Hostname",
+        },
+        {
+          :component => "text-field",
+          :name      => "endpoints.events.port",
+          :label     => "AMQP Port",
+          :type      => "number"
+        },
+        {
+          :component => "text-field",
+          :name      => "endpoints.events.username",
+          :label     => "AMQP Username"
+        },
+        {
+          :component => "text-field",
+          :name      => "endpoints.events.password",
+          :label     => "Password",
+          :type      => "AMQP password"
+        }
+      ]
+    }.freeze
+  end
+
+  # Verify Credentials
+  # args:
+  # {
+  #   "endpoints" => {
+  #     "default" => {
+  #       "server"   => nil,
+  #       "port"     => nil,
+  #       "username" => nil,
+  #       "password" => nil,
+  #     },
+  #     "events"  => {
+  #       "server"   => nil,
+  #       "port"     => nil,
+  #       "username" => nil,
+  #       "password" => nil,
+  #     }
+  #   }
+  # }
+  def self.verify_credentials(args)
+    default_endpoint = args.dig("endpoints", "default")
+    server, port, username, password, api_version = default_endpoint&.values_at(
+      "server", "port", "username", "password", "api_version")
```

Comment:
There's no api_version in the DDF form.

Reply:
:+1: good catch

Reply:
Added with an initialValue of 5.5 https://github.com/ManageIQ/manageiq-providers-vmware/pull/457/files#diff-5f06ba425c7637143676aadd6dda749bR68